Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -1301,20 +1312,31 @@ static _WKWebExtensionError toAPI(WebExtension::Error error)
 
     populateBackgroundPropertiesIfNeeded();
 
-    if (m_backgroundServiceWorkerPath)
+    if (m_backgroundServiceWorkerPath || m_backgroundPagePath)
         return nil;
```

Comment:
Does this mean that the content has already been generated?

Reply:
It means we shouldn't generate content since we have paths. These could be ASSERTs instead, but an extension can still give here if they try to load the generated page / script path.